standards/imaging/1.0.0/imaging_schema.md

```diff
@@ -0,0 +1,402 @@
+  * Deprecating metadata fields
+  * Changing the type or format of a metadata field
+ 
+**Minor version** is incremented when additive schema updates are introduced:
+  * Adding metadata fields
+  * Changing the validation requirements for a metadata field
+  
+**Patch version** is incremented for editorial updates.
+
+All changes are documented in the schema [Changelog](#appendix-a-changelog).
+
+
+## Overview
+The following descriptive metadata are associated with all imaging datasets to ensure that datasets can be found by searching for common characteristics of datasets. These metadata MUST be programmatically validated to ensure compliance. Additional metadata MAY be annotated at the discretion of data stewards.
+
+**Categories of metadata:**
+- **Data description** — dataset-level metadata (source, license, compliance).  
+- **Subject** — species, genotype, sex, age.  
+- **Procedure** — pre-acquisition treatments, labeling, specimen preparation.  
+- **Instrument** — imaging device metadata.  
+- **Acquisition** — acquisition software and hardware configuration.  
+- **Channel description** — per-channel settings.  
+<br>
+
+For data in Zarr format, the metadata defined by [OME-Zarr](https://ngff.openmicroscopy.org/0.1/) SHOULD be implemented.<br>
+<br>
+
+**For general images:**
+- All metadata are stored at the OME-Zarr root level. The metadata can be stored in separate json file(s) unless specified in OME-Zarr.
+
+**For high content screening (HCS):**
+- All metadata are stored at the OME-Zarr root level. The metadata can be stored in separate json file(s) unless specified in OME-Zarr.
+- Plate-level metadata: data description, instrument, acquisition.  
+- Well-level metadata: subject, procedure, channel description.
+
+**For CryoET data:**
+- The full set of metadata for CryoET data is described [here](TBD). <br>
+
+The mapping of CryoET data and DCA data to the XMS is specified [here](https://github.com/chanzuckerberg/data-guidance/blob/main/standards/imaging/1.0.0/XMS_mapping.md).
+
+## Data description
+Administrative metadata about the source of the data, funding, relevant licenses, and restrictions on use.
+
+Cross-modality metadata: 
+Security & Compliance Metadata and Technical Metadata outlined in the [Data registration needs](https://czi.atlassian.net/wiki/spaces/SCI/pages/3896868957/Data+Registration+Needs+Metadata) are captured here.
+
+### imaging_method
+<table><tbody>
+    <tr>
+      <th>Key</th>
+      <td>imaging_method</td>
+    </tr>
+    <tr>
+      <th>Description</th>
+      <td>Defines the imaging method that was used to create the dataset</td>
+    </tr>
+    <tr>
+      <th>Annotator</th>
+      <td>Submitter MUST annotate.</td>
+    </tr>
+    <tr>
+      <th>Value</th>
+        <td><code>List[String]</code>. 
+         The List element MUST be a term in <a href="https://webprotege.stanford.edu/#projects/ee9f28eb-bcfe-469d-a9b4-48929f100eec/edit/Classes">the imaging methods list </a>.
```

Comment:
This link prompted me for a login.

Reply:
Will update to this doc link:
https://docs.google.com/document/d/1v34rJ2bGqF_6BUMLd2zvu36AReTmRu0dIxMCNhyqTYs/edit?tab=t.iuu4anexqv55#heading=h.l62d2b5ana